src/tests/contracts/HR721/abi.ts

```diff
@@ -0,0 +1,333 @@
+export const ABI: any = [
```

Comment:
Delete this file is already in [HRC721](https://github.com/blockcoders/harmony-marketplace-sdk/blob/main/src/tests/contracts/HRC721/abi.ts)

Reply:
Copy that!